src/lib/InstantSearch.js

```diff
@@ -205,9 +205,9 @@ class InstantSearch extends EventEmitter {
       if (nextState) {
         // We don't want to re-add URlSync `getConfiguration` widget
         // it can throw errors since it may re-add SearchParameters about something unmounted
-        this.searchParameters = this.widgets
-          .filter(w => w.constructor.name !== 'URLSync') // TODO: check the routing for this
```

Comment:
Just to let you know, we can't use the `name` attribute of function or constructor. Once Uglify pass on the code all values are renamed and the constructor name is changed (except with an Uglify option that is by default to `false`, see [`keep_fnames `](https://github.com/mishoo/UglifyJS2#minify-options)).

Reply:
That's good to know thanks. It might have never worked in production then for the `URLSync`.

Reply:
Yep indeed.